.circleci/integration-tests/Dockerfile

```diff
@@ -14,6 +14,18 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 RUN unzip awscliv2.zip
 RUN ./aws/install
 
+# install eksctl
+RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+RUN mv /tmp/eksctl /usr/local/bin
+RUN eksctl version
+
+# install kubectl
+RUN curl -o kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.22.6/2022-03-09/bin/linux/amd64/kubectl
+
+RUN chmod +x ./kubectl
+RUN mv ./kubectl /usr/local/bin
```

Comment:
In my opinion, we can also combine these 3 RUN commands into a single RUN command using `&&` to reduce the number of layers.

Reply:
Added as single line